.github/workflows/build-deploy-dev.yml

```
@@ -0,0 +1,190 @@
+          ${{ env.PROJECT_NAME }} Build Started 🚀
+          📁 Repository: ${{ github.repository }}
+          👤 Initiator: ${{ github.actor }}
+          🌿 Branch: ${{ github.ref_name }}
+          🕒 Timestamp: ${{ github.event.head_commit.timestamp}} 
+          📝 Commit Message: ${{ github.event.head_commit.message }}"
+
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Setup Go Environment
+        uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)